### ISSUE TITLE
provide unit tests for the new discovery mechanism

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1880]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1876
- Created at 2013-04-10 15:54:12 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:20:16 as wontfix
- Assigned to nobody
---

Recently as part of developing ticket #1032 a lot of new site discovery mechanisms including a pluggable systems were introduced. We should add unit tests for this functionality.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-04-11 15:50:48](https://pagure.io/SSSD/sssd/issue/1876#231401)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.10.0
rhbz:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-05-13 15:40:59](https://pagure.io/SSSD/sssd/issue/1876#231402)**

Fields changed

milestone: SSSD 1.10.0 => SSSD 1.11 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-05-21 00:48:27](https://pagure.io/SSSD/sssd/issue/1876#231403)**

Fields changed

blocking:  => 1923

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-06-28 23:55:14](https://pagure.io/SSSD/sssd/issue/1876#231404)**

Moving open tickets from 1.11 beta to 1.11 beta2

milestone: SSSD 1.11 beta => SSSD 1.11 beta 2

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-07-24 14:54:05](https://pagure.io/SSSD/sssd/issue/1876#231405)**

Fields changed

changelog:  => 
milestone: SSSD 1.11 beta 2 => Interim Bucket

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-07-30 12:54:43](https://pagure.io/SSSD/sssd/issue/1876#231406)**

Fields changed

milestone: Interim Bucket => SSSD 1.12 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-02-19 11:52:55](https://pagure.io/SSSD/sssd/issue/1876#231407)**

Fields changed

blocking: 1923 => #1923

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-05-30 15:53:20](https://pagure.io/SSSD/sssd/issue/1876#231408)**

Fields changed

milestone: SSSD 1.12 beta => SSSD 1.12.1

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-09-08 20:09:27](https://pagure.io/SSSD/sssd/issue/1876#231409)**

Mass-moving all tickets that didn't make 1.12.1 into 1.12.2

milestone: SSSD 1.12.1 => SSSD 1.12.2

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-09-30 19:06:43](https://pagure.io/SSSD/sssd/issue/1876#231410)**

We need to do a release as requested by downstream. Moving tickets that are not fixed already or very close to acking to 1.12.3

milestone: SSSD 1.12.2 => SSSD 1.12.3

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-11-19 18:13:09](https://pagure.io/SSSD/sssd/issue/1876#231411)**

It makes sense to write the unit tests now instead of moving them to a later release again, but the tickets in 1.12.3 should be more focused at this point. Moving to 1.12.4 

mark:  => 0
milestone: SSSD 1.12.3 => SSSD 1.12.4

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-17 21:06:06](https://pagure.io/SSSD/sssd/issue/1876#231412)**

Some tests are on the list in scope of #1884. During the 1.13 scoping we agreed to defer unit tests for functionality that has been released some time ago and not changed recently at the same time.

milestone: SSSD 1.12.4 => SSSD Deferred

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:43:44](https://pagure.io/SSSD/sssd/issue/1876#231413)**

**Metadata Update from @jhrozek**:
- Issue marked as blocked by: #1923
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:20:15](https://pagure.io/SSSD/sssd/issue/1876#636909)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:20:16](https://pagure.io/SSSD/sssd/issue/1876#636910)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
